packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructService.ts

```diff
@@ -315,6 +317,7 @@ class IndigoService implements StructService {
       const commandOptions: CommandOptions = {
         ...this.defaultOptions,
         ...options,
+        'output-content-type': 'application/json',
```

Comment:
I suggest provide detailed description as per PR template. 
My main concern here is that you're changing the logic, which worked before w/o explaining what was changed

Reply:
done